forem/forem
# BETA version of the Storybook Multi Input

## What type of PR is this? (check all applicable)


## Description

I created an MVP of the Multi Input Component. It does not yet cover all of the functionality required, but I'd like to merge this one in and then build upon it.

I would still need to add:
- validation 
- allow the regex for an input to be passed through as a prob
- add tests 
- add documentation
- delete an item using keyboard navigation (accessibility)

## Related Tickets & Documents

- Related Issue #https://github.com/forem/forem/issues/17577

## QA Instructions, Screenshots, Recordings

- Locally run yarn install && yarn storybook
- Navigate to the BETA > MultiInput story
- Play around with adding and removing items (email addresses).

Demo Video:
https://www.loom.com/share/4f7a9e49bc3e4739a4a7e20453389488

It should look somewhat like this, however in the next PR I will focus on polishing it up.
<img width="269" alt="Screenshot 2022-06-06 at 15 14 46" src="https://user-images.githubusercontent.com/2786819/172167799-468f7076-af3a-44db-8764-59b147b56727.png">

### UI accessibility concerns?

I tested keyboard navigation and voice over, but could have potentially missed a few things out.
 
## Added/updated tests?

- [x] No, and this is why: I'll add tests once I polish up the BETA version

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Storybook](https://storybook.forem.com/) (for Crayons components)
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
      or in a [forem.dev](http://forem.dev) post
- [x] This change does not need to be communicated right now

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![alt_text](https://media.giphy.com/media/SwImQhtiNA7io/giphy.gif)
